src/EdgeAnalysis.jl

```diff
@@ -606,7 +606,45 @@ function identifyobjects!(sim,corespcsinds,corerxninds,edgespcsinds,
         tempnewobjectvals = Array{Float64,1}()
         tempnewobjecttype = []
     end
-    
+
+
+    if length(transitorydict) > 0 && !firsttime
+        transitoryoutdict = Dict()
+        TS = transitorysensitivitiesfulltrapezoidal(sim,t;normalized=false)
+        TS .*= sim.p'
+        TS = TS[:,length(sim.names)+1:end]
```

Comment:
Could you add @views here

Reply:
I don't think @views will work here in the only line with a slice we're defining a new matrix so it has to create a new matrix. I have added an @views to the tsscale line.